src/components/DataGridWithInfiniteScroll/Constants.tsx

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright (c) 2022 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+type typeConstants = {
+    COLUMN_SELECTION_MENU_TOP: number; COLUMN_SELECTION_MENU_LEFT: number; CHANGE_SELECT_MENU_TOP: number; CHANGE_SELECT_MENU_LEFT: number; RESET_SETTINGS: string; EMPTY_DATA_GRID_NO_ITEMS_FOUND: string; ID_FOR_SORTING: string; DEFAULT_DATAGRID_BODY_HEIGHT: number;LOADING_INFINITE_SCROLL:string;COLUMN_PICKER:string;CLOSE:string;
+}
+export const Constants: typeConstants = {
+    COLUMN_SELECTION_MENU_TOP: 50,
+    COLUMN_SELECTION_MENU_LEFT: 180,
+    CHANGE_SELECT_MENU_TOP: 50,
+    CHANGE_SELECT_MENU_LEFT: 215,
+    RESET_SETTINGS: 'Reset Settings',
+    EMPTY_DATA_GRID_NO_ITEMS_FOUND: 'No items found!',
+    ID_FOR_SORTING: 'allSelect',
+    DEFAULT_DATAGRID_BODY_HEIGHT: 550,
+    LOADING_INFINITE_SCROLL:'Loading...',
```

Comment:
```suggestion
    DEFAULT_LOADING_INFINITE_SCROLL_TEXT:'Loading...',
```
allow user to change it if required.

Reply:
Fixed.